.gitignore

```diff
@@ -4,3 +4,6 @@ tmp
 docs/build
 .ruby-version
 node_modules
+
+.idea
+ember-simple-auth.iml
```

Comment:
IMHO this shouldn't be part of the project `.gitignore` file. Use a [global `.gitignore` file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) instead.